pcap-common.c

```diff
@@ -1228,7 +1228,14 @@
  */
 #define LINKTYPE_ZBOSS_NCP	292
 
-#define LINKTYPE_MATCHING_MAX	292		/* highest value in the "matching" range */
+/*
+ * Ultra-wideband (UWB) controller interface protocol (UCI).
+ * Protocol description on the FiRa consortium website:
+ *  https://groups.firaconsortium.org/wg/members/document/1679
```

Comment:
This links to a members only document.  Is there a document that is public?

Reply:
Sadly none that I am aware of...

Reply:
So do you have a publicly-available description of the protocol?

Reply:
The FiRa has not made the specification public. 
Please check out the discussion on https://github.com/the-tcpdump-group/tcpdump-htdocs/pull/28; 
I added the packet framing description; still waiting on review for this.

Reply:
The link type 299 was approved on tcpdump htdocs. @guyharris Can you pickup the review for this MR ? Thanks !